types/lodash/lodash-tests.ts

```diff
@@ -726,7 +726,7 @@ _.chain([1, 2, 3, 4]).unshift(5, 6); // $ExpectType CollectionChain<number>
     // $ExpectType AbcObject[]
     _.intersectionWith(...[list, list], (a, b) => {
         a; // $ExpectType AbcObject
-        b; // $ExpectType never
+        b; // $ExpectType never || AbcObject
```

Comment:
TIL that `||` is allowed in those comments, wild!

Reply:
Yes, it's a dtslint thing, so we can ExpectType multiple things. Sorta a meta-level union.